bark/runtime/viewer/viewer.py

```diff
@@ -289,6 +289,47 @@ def drawLabelsAsText(self, observed_world, label_functions, evaluator_type):
         self.drawText(position=(0.7, 0.9), text=str)
 
     def drawWorld(self, world, eval_agent_ids=None, filename=None, scenario_idx=None, debug_text=True):
+      if eval_agent_ids:
+          sensed_world = world.agents[eval_agent_ids[0]].sensed_world
+          self.drawTrueWorld(world, eval_agent_ids)
+          self.drawSensedWorld(sensed_world, eval_agent_ids, filename, scenario_idx, debug_text)
+      else:
+          self.drawWorldImplementation(world, eval_agent_ids, filename, scenario_idx, debug_text)
+
+    def drawSensedWorld(self, sensed_world, eval_agent_ids=None, filename=None, scenario_idx=None, debug_text=True):
+      self.drawWorldImplementation(sensed_world, eval_agent_ids, filename, scenario_idx, debug_text)
+
+    def drawTrueWorld(self, world, eval_agent_ids):
+      # draw only boundaries of agents in same color as observed agents
+      for i, (agent_id, agent) in enumerate(world.agents.items()):
+          color_line, color_face, alpha = self.GetAgentColor(agent, eval_agent_ids)
+          color_face = (1.0, 1.0, 1.0) # face color white
```

Comment:
why set a default color here?

Reply:
we ground truth shape should only have borders drawn and color white. I did not want to introduce another parameter when it does not make sense for me.